src/port1.0/portconfigure.tcl

```diff
@@ -66,6 +66,17 @@ default configure.objcxxflags           {${configure.cxxflags}}
 default configure.universal_cxxflags    {[portconfigure::configure_get_universal_cflags]}
 default configure.universal_objcxxflags {${configure.universal_cxxflags}}
 
+# helper function to set configure.cxx_stdlib
+proc portconfigure::configure_cxx_stdlib {} {
+    global cxx_stdlib compiler.cxx_standard
+
+    if {${cxx_stdlib} eq "libstdc++" && ${compiler.cxx_standard} >= 2011} {
```

Comment:
For configurations that default to libstdc++, this will result in some projects using libstdc++ and others using macports-libstdc++.  This is fine as long as there are no objects being handed off across that boundary.

I worry about libraries that require C++11 for their implementation but expose an API boundary that works fine with older language versions, so the library itself might build with macports-libstdc++ while the client that is using it builds with libstdc++.

Reply:
We already have exactly this situation in the cxx11 1.1 portgroup. I brought up this objection multiple times, but I was told that we believed that this was not a problem because we were using `-D_GLIBCXX_USE_CXX11_ABI=0`. Therefore, the cxx11 1.1 portgroup was put into widespread use. If that flag is not sufficient to solve the problem, then we already have tons of published ports subject to this problem.

Reply:
Your point is well taken.
When this problem occurred in [wxWidgets](https://github.com/macports/macports-ports/commit/963e8f7ba330899e9fc0cd1313e0af6d7ae71198), the cause of the runtime crash was not at all easy to see at first.
Setting the default C++ requirement to [2011](https://github.com/macports/macports-base/pull/88/commits/bf864905e12b8489c7be8d1413d0bd5b9d82761c) might mitigate this problem.
Any ports that are confident that this is not a problem can set `compiler.cxx_standard` to a lower value.